### PR TITLE
Detect console.logs and its equivalent in other languages

### PR DESCRIPTION
**Pull request by baristaGeek**
## Description
The goal of this PR is to detect `console.log()`s and its equivalent in other languages (`System.out.println()`, `println()`, `print()`, `fmt.Println()`, etc.). The initial goal is to successfully detect such. Then we can comment them and add them to the PR Rating score. 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)  
- [x] New feature (non-breaking change which adds functionality)  
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)  
- [ ] Documentation update  
- [ ] Chore: cleanup/renaming, etc  
- [ ] RFC  

## Notes
I am initially sending this PR with a simple `console.log()` to test this PR.

## Acceptance
- [x] I have read [the Contributing guidelines](CONTRIBUTING.md)
- [x] I have read the [Code of Conduct](CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by EstebanDalelR — COMMENTED** [review]
This is a test, right?
Does not seem to work.

**Comment by baristaGeek** [comment]
> This is a test, right? Does not seem to work.

As stated in the description, yes

**Comment by baristaGeek** [comment]
@EstebanDalelR pls retest, I added some PRs with hello worlds in some languages to do so. A few notes:

1. Commenting the specific line diff would be a next step. I tried doing so with `octokit.rest.pulls.createReviewComment` (its equivalent on our `octokit.request("....") `fashion but couldn't achieve it. Right now we're adding an additional comment if console logs are detected
2. I tried changing the function from a void that comments to a funciton that returns a boolean but I wasn't successful with that because the call to openai is awaited and if console logs are detected, that would be later and therefore a false would be returned despite console logs being there.
3. Modifying the label to "don't merge" would be a remaining step to send this to prod, but let's keep testing it. I'm expecting testing this feature to be extensive. 
4. The name of the file is `detectConsoleLogs.ts` which is what it's intended for at the moment. However we can and should change that to something of a broader scope in the future.

File: utils/actions/detectConsoleLogs.ts [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  diffFiles.map(async (file) => {
+    const additions = getAdditions(file.patch ?? "");
+
+    const consoleLogDetectionPrompt = `This is a list of code additions. Identify 
+    if there's a console log or its equivalent in another programming language 
+    (console.log(), println(), System.out.println(), print(), fmt.Println(), etc.). 
+    If the console log or its equivalent in another language is in a code comment, don't
+    count it as a detected console log. For example JavaScript comments start with // or /*, 
+    Python comments start with #.
+    Other console functions such as console.info() should'n't be counted as console logs.
```
Comment (EstebanDalelR) [inline utils/actions/detectConsoleLogs.ts:104]:
```suggestion
    Other console functions such as console.info() shouldn't be counted as console logs.
```

File: utils/actions/detectConsoleLogs.ts [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        .createChatCompletion({
+          model: "gpt-3.5-turbo-16k",
+          messages: [
+            {
+              role: "system",
+              content: `${consoleLogDetectionPrompt} \n ${additions}`,
+            },
+          ],
+        })
+        .then((result) => {
```
Comment (EstebanDalelR) [inline utils/actions/detectConsoleLogs.ts:120]: Could we Promise.all this?

**Review by EstebanDalelR — APPROVED** [review]

File: utils/actions/detectConsoleLogs.ts (hunk 2 above)
Comment (baristaGeek) [reply utils/actions/detectConsoleLogs.ts:120]: Absolutely! Great suggestion.

**Pull request merged**